From 605566d38eed104b9e5ece9870ff228488e0d269 Mon Sep 17 00:00:00 2001
From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 13:13:31 -0700
Subject: ignore userwarning in tests

 Required to run tests from source with the package already installed.
 Else one gets check_version_conflict warning from pkg_resources.
Patch-Name: ignoreuserwarning.patch
---
 tornado/test/runtests.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tornado/test/runtests.py b/tornado/test/runtests.py
index b81c5f2..b4b2569 100644
--- a/tornado/test/runtests.py
+++ b/tornado/test/runtests.py
@@ -102,6 +102,7 @@ def main():
     # setuptools sometimes gives ImportWarnings about things that are on
     # sys.path even if they're not being used.
     warnings.filterwarnings("ignore", category=ImportWarning)
+    warnings.filterwarnings("ignore", category=UserWarning)
     # Tornado generally shouldn't use anything deprecated, but some of
     # our dependencies do (last match wins).
     warnings.filterwarnings("ignore", category=DeprecationWarning)
